src/js/tech/html5.js

```diff
@@ -200,6 +200,46 @@ class Html5 extends Tech {
     });
   }
 
+  /**
+   * Attempt to force override of native audiio/video tracks.
+   *
+   * @param {Boolean} override - If set to true native audio/video will be overridden,
+   * otherwise native audio/video will potentially be used.
+   */
+  overrideNativeTracks(override) {
+    // If there is no behavioral change don't add/remove listeners
+    if (override === this.override_) {
```

Comment:
could we compare this against the values of `featuresNativeVideoTracks` and `featuresNativeAudioTracks` instead? that way we don't have another flag to worry about.

Reply:
Given a brief chat with @gkatsev I'll set this to check against `featuresNativeVideoTracks && featuresNativeAudioTracks`.